src/app/src/main/assets/mock_json/200_success_details_departures.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
I noticed that all these json mock files are only being used by `MockAppGluService`, which is located within `androidTest`. Shouldn't they be located there as well? Was there any reason for them to be part of the main source code?